config/vmagent-prometheus.yml.erb

```diff
@@ -24,3 +24,10 @@ scrape_configs:
   - job_name: <%= ENV['VMAGENT_APP_JOB_NAME'] || 'web' %>
     static_configs:
     - targets: ['localhost:<%= ENV['PORT'] %>']
+
+  - job_name: instance
+    metric_relabel_configs:
```

Comment:
I think this should be on the app job above rather than a new job

Reply:
If I set it like I do environment will it override it? or do I need something else for it?

Reply:
I'm going to defer to @JamesFrost for his input on this, since this is new to me. James, we'd just like to relabel "instance" to be the same as "dyno" in our metrics.

Currently we get something like:

```
control_api_requests_total{account_id="q-zBlQ", dyno="web.1", environment="website-web-1747-contro-73ind3", instance="localhost:58027", job="web"}
```

and we'd prefer:

```
control_api_requests_total{account_id="q-zBlQ", dyno="web.1", environment="website-web-1747-contro-73ind3", instance="web.1", job="web"}
```

since `localhost:58027` is pretty meaningless to us.

First of all, is this a reasonable thing to do? And if so, how should we do it?

Reply:
Yes, overriding the instance label is reasonable.

I believe it can be done as part of the existing job, as @lmars suggested.

As it stands, I don't know if this suggested change will do anything, as the new job doesn't have any targets? The new job won't be collecting any metics, and so no relabelling will take place. The other jobs will collect as normal, with the instance label being set as before.

Is overriding the instance label to have the same value as the existing dyno label slightly redundant? If it's overriden for one job, should we override it for all jobs?

Reply:
thanks @JamesFrost, @lmars what do you think to dropping the metric? I can adapt this PR to remove it instead?

Reply:
If you're suggesting removing the instance label entirely, I think this is wrong - it's expected to exist.

Reply:
I see, what do you think the best practice is? is there a more useful metric we can make it other than dyno? I agree with the redundancy of making it the same as dyno, but not sure the best approach.

Reply:
Best practice is that this label uniquely identifies a target within a job.

I don't have any strong opinions about what the specific value of this label should be. 

Based on the conversation so far, it sounds like we don't have a particularly compelling reason to override this label.

Reply:
Thanks for your input @JamesFrost, yes this seems like a distraction. @denissellu let's just leave the instance label as is and close this PR.

Reply:
👍